app/helpers/application_helper.rb

```diff
@@ -20,4 +22,16 @@ def markdown(text)
     redcarpet = Redcarpet::Markdown.new(renderer, extensions)
     (redcarpet.render text).html_safe
   end
+
+  def geocodable?(user)
+    user && user.location.is_geocodable?
+  end
+
+  # miles between a specific dish and given lat/lng location
+  def dish_distance(dish, lat, lon)
```

Comment:
There's a few calls on `dish` in this method, perhaps this is a responsibility that belongs in `Dish`. Consider moving this method to `Dish`.